Taskfile.yaml

```diff
@@ -198,6 +205,13 @@ tasks:
         vars:
           CLI_ARGS: add-dev-user
 
+  docker:rbac:loaddata:
+    desc: "Load RBAC user and roles from tools/initial_data.yml file"
+    cmds:
+      - task: dev
+        vars:
+          CLI_ARGS: load-rbac-data
+
```

Comment:
This is not needed, isn't?

Reply:
agreed

Reply:
You have already duplicated them: `docker:rbac:loaddata` and `minikube:rbac:loaddata` which is already implemented in this PR. That's why I asked about `docker:rbac:loaddata` Which actually does not do nothing I think. It could be for docker-compose, but as docker compose is the only way to perform a regular deployment I think it's fine as it is, without seeding the initial_data.